packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -283,11 +286,13 @@ const Tab = ({
       aria-disabled={isDisabled}
       className={variant === 'framework' ? paneClassNames : tabClassNames}
       title={label}
-      aria-label={isClosable ? `${label}. ${tabDeleteLabel}` : label}
+      aria-label={label}
+      aria-describedby={isClosable ? deleteResponseId : undefined}
```

Comment:
let's return `null` instead of `undefined`.

Reply:
We should also add a unit test for this change. E.g.

https://github.com/cerner/terra-framework/blob/698d1309f6a95f2f1737477138809c2bf6985a8f/packages/terra-data-grid/tests/jest/ColumnResizeHandle.test.jsx#L129-L142